README.md

```diff
@@ -62,11 +72,19 @@ Add `tmphabricator` to your Phabricator `load-libraries`:
 
 # Generating the library map
 
-When developing a new module it needs to be referenced in some auto-generated files. To
-generate these:
+The modules in this repository need to be referenced in some auto-generated
+files. To generate them, ensure you have followed the installation instructions
+above, then run `arc liberate` from the root of this repository:
+
+```
+cd tmphabricator
+arc liberate
+# Should output "Done."
+```
 
-1. clone https://github.com/phacility/phabricator to the same location as tmphabricator
-1. navigate into the tmphabricator directory
-1. run `arc liberate`
+**NOTE:** `arc liberate` automatically attempts to build `libxhpast` in
+`/opt/libphutil/support/xhpast/` if it detects that it doesn't already exist.
```

Comment:
Bears noting that libphutil has gone away upstream; we haven't updated to that version yet but presumably will at some point.